### PR TITLE
Modify rule S6275: Add description

### PR DESCRIPTION
**Pull request by hendrik-buchwald-sonarsource**
Implementation ticket: SONARIAC-125

### CODE REVIEW

File: rules/S6275/description.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+Amazon Elastic Block Store (EBS) is a block-storage service for Amazon Elastic Compute Cloud (EC2). EBS volumes can be encrypted, ensuring the security of both data-at-rest and data-in-transit between an instance and its attached EBS storage. Currently, encryption is disabled.
```
Comment (eric-therond-sonarsource) [inline rules/S6275/description.adoc:1]: I would change:
`Currently, encryption is disabled.`
with something similar to:
`Newly created EBS volumes are automatically encrypted if the "encrypted by default" feature is enabled.`
with a link to: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html#encryption-by-default

File: rules/S6275/cloudformation/rule.adoc [hunk 2]
```diff
@@ -1,74 +1,42 @@
...
-            "DeletionPolicy" : "Snapshot"
-	    }
-    }
-}
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  Ec2Volume:
+    Type: AWS::EC2::Volume
+    Properties:
+      # Sensitive: no "Encrypted" entry provided
```
[the remark this answers is not in the document]
Comment (eric-therond-sonarsource) [reply rules/S6275/cloudformation/rule.adoc:26]: same thing for the similar terraform noncompliant codes

**Review by eric-therond-sonarsource — APPROVED** [review]
I put some minor suggestions

File: rules/S6275/cloudformation/rule.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (eric-therond-sonarsource) [reply rules/S6275/cloudformation/rule.adoc:26]: but it's depend on where you want to raise the issue

[the remark this answers is not in the document]
Comment (hendrik-buchwald-sonarsource) [reply rules/S6275/cloudformation/rule.adoc:26]: Makes sense. When the value is set to `false` I would raise the issue there but if no property is provided that does not work. So in that case raising at the definition of the resource makes the most sense.

**Pull request merged**